setlexsem/generate/generate_sets.py

```diff
@@ -208,16 +209,9 @@ def make_hps_set(
 
 def get_sampler(hp: Dict[str, Any], random_state: random.Random) -> Sampler:
     set_type = hp["set_types"]
-
     if set_type == "numbers":
-        # override n if item-length is defined
-        if hp.get("item_len"):
-            n = None
-        else:
-            n = hp["n"]
-
         sampler = BasicNumberSampler(
-            n=n,
+            n=hp["n"] if hp.get("n") else None,
```

Comment:
I believe line 214 can be just:
```
n=hp.get("n")
```
as `dict.get(key)` returns `None` if the key is not present in the dictionary.

Reply:
Nice! fixed it!